src/components/NewArticleComments.jsx

```diff
@@ -0,0 +1,28 @@
+import axios from "axios"
+import { useEffect, useState } from "react"
+import { useParams } from "react-router-dom"
+
+function NewArticleComments(){
+    const article = useParams()
+    const articleChoice = article.article_id
+    const commentAuthor = "grumpy19"
```

Comment:
This would be much better if it were kept higher up in the app and passed by props or, ideally, context.